mappings/net/minecraft/world/chunk/Chunk.mapping

```diff
@@ -0,0 +1,129 @@
+CLASS net/minecraft/class_43 net/minecraft/world/chunk/Chunk
+	FIELD field_954 blocks [B
+	FIELD field_956 world Lnet/minecraft/class_18;
+	FIELD field_957 meta Lnet/minecraft/class_257;
+	FIELD field_958 skyLight Lnet/minecraft/class_257;
+	FIELD field_959 blockLight Lnet/minecraft/class_257;
+	FIELD field_960 heightmap [B
+	FIELD field_961 minHeightmapValue I
+	FIELD field_962 x I
+	FIELD field_963 z I
+	FIELD field_964 blockEntities Ljava/util/Map;
+	FIELD field_965 entities [Ljava/util/List;
+	METHOD <init> (Lnet/minecraft/class_18;II)V
+		ARG 1 world
+		ARG 2 x
+		ARG 3 z
+	METHOD <init> (Lnet/minecraft/class_18;[BII)V
+		ARG 1 world
+		ARG 2 blocks
+		ARG 3 x
+		ARG 4 z
+	METHOD method_858 (II)Z
+		ARG 1 x
+		ARG 2 z
+	METHOD method_859 getBlockId (III)I
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_860 setBlockId (IIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 rawId
+	METHOD method_861 setBlock (IIIII)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 rawId
+		ARG 5 meta
+	METHOD method_862 setBlockEntity (IIILnet/minecraft/class_55;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockEntity
+	METHOD method_863 getSlimeRandom (J)Ljava/util/Random;
+		ARG 1 scrambler
```

Comment:
the scrambler in modern is only a part of the random seed, i suggest using random for consistency with other Random arguments in the mappings

Reply:
Agreed with Dany here, scrambler is different from a full copy of the random seed.

Reply:
It is exactly scrambler from modern versions though.
![image](https://github.com/calmilamsy/biny-mappings/assets/19170211/a4acf562-c192-4589-bce6-417b7a0b4109)
![image](https://github.com/calmilamsy/biny-mappings/assets/19170211/cfd2a77d-e245-495b-97c8-6518006e632a)

Reply:
sorry, my fault, dont know what ive been smoking, youre right its only a part of the randomSeed